webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/errors/ErrorMessage.java

```diff
@@ -26,13 +26,13 @@
 @XmlRootElement
 public class ErrorMessage {
 
-  /** Status code = 400, 405, 500 etc */
+  // Status code = 400,404,500 etc.
   private int errorStatusCode;
 
-  /** Basic Description for the Error = InternalServerException etc. * */
+  // Basic Description for the Error = InternalServerException etc.
   private String errorType;
 
-  /** More Information regarding the thrown exception = Reason for exception in Server side * */
+  // More Information regarding the thrown exception = Reason for exception in Server side
```

Comment:
Javadoc comments should start with `/**`. i.e no need to change these line (check above lines as well).